### PR TITLE
Initial documentation for cover templates

### PR DESCRIPTION
**Pull request by PhracturedBlue**
**Description:**
Documentation for cover templates

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#8100

### CODE REVIEW

File: source/_components/cover.template.markdown [hunk 1]
```diff
@@ -0,0 +1,77 @@
+---
+layout: page
+title: "Template Light"
```
Comment (Landrash) [inline source/_components/cover.template.markdown]: Should be cover.

File: source/_components/cover.template.markdown [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+description: "Instructions how to integrate Template lights into Home Assistant."
```
Comment (Landrash) [inline source/_components/cover.template.markdown]: Should be cover.

File: source/_components/cover.template.markdown [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+date: 2016-05-18 20:32
```
Comment (Landrash) [inline source/_components/cover.template.markdown]: Update date to actual date of creation?

File: source/_components/cover.template.markdown [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+comments: false
+sharing: true
+footer: true
+ha_category: Cover
+ha_release: 0.49
+ha_iot_class: "Local Push"
+logo: home-assistant.png
+---
+
+The `template` platform creates covers that combine components and provides the ability to run scripts or invoke services for each of the open, close, stop, position, and tilt commands of a cover.
```
Comment (Landrash) [inline source/_components/cover.template.markdown]: The `template`platform **can** create.

File: source/_components/cover.template.markdown [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+          service: script.open_garage_door
+        close_cover:
+          service: script.close_garage_door
+        stop_cover:
+          service: script.stop_garage_door
+```
+
+Configuration variables:
+
+- **coverss** array (*Required*): List of your coverss.
```
Comment (Landrash) [inline source/_components/cover.template.markdown]: **cover** instead of **coverss**?

File: source/_components/cover.template.markdown [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+  - **friendly_name** (*Optional*): Name to use in the Frontend.
+  - **value_template** (*Optional*): Defines a [template](/topics/templating/) to get the state of the cover. Legal valules are open/true or closed/false. Either value_template or position_template must be specified.
+  - **position_template** (*Optional*): Defines a [template](/topics/templating/) to get the state of the cover. Legal values are numbers between 0 (closed and 100 (open). Either value_template or position_template must be specified.
```
Comment (Landrash) [inline source/_components/cover.template.markdown]: *(closed* is missing closing **)**.

File: source/_components/cover.template.markdown [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+  - **icon_template** (*Optional*): Defines a [template](/topics/templating/) to specify which icon to use. Either value_template or position_template must be specified.
+  - **open_cover** (*Required*): Defines an [action](/getting-started/automation/) to run when the cover is opened.
+  - **close_cover** (*Required*): Defines an [action](/getting-started/automation/) to run when the cover is closed.
+  - **stop_cover** (*Required*): Defines an [action](/getting-started/automation/) to run when the cover is stopped.
+  - **set_cover_position** (*Optional*): Defines an [action](/getting-started/automation/) to run when the cover is set to a specific value (between 0 and 100).
+  - **set_cover_tilt_position** (*Optional*): Defines an [action](/getting-started/automation/) to run when the cover tilt is set to a specific value (between 0 and 100).
+  - **entity_id** (*Optional*): Add a list of entity IDs so the switch only reacts to state changes of these entities. This will reduce the number of times the light will try to update it's state.
```
Comment (Landrash) [inline source/_components/cover.template.markdown]: **light** instead**cover**.

File: source/_components/cover.template.markdown [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+
+
+## {% linkable_title Examples %}
+
+In this section you will find some real life examples of how to use this light.
```
Comment (Landrash) [inline source/_components/cover.template.markdown]: **Light** instead of **cover**

**Comment by PhracturedBlue** [comment]
Thank you for taking the time to review this

**Comment by Landrash** [comment]
@PhracturedBlue No problem. Looks good and can be merged when parent PR is merged.

**Pull request merged**